versioned_docs/version-10.x/broadcasting.md

```
@@ -0,0 +1,1240 @@
+# 브로드캐스팅 (Broadcasting)
+
+- [소개](#introduction)
+- [서버 사이드 설치](#server-side-installation)
+    - [설정](#configuration)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)